src/endpoint/invites.ts

```diff
@@ -0,0 +1,60 @@
+import { EndpointClient, EndpointClientConfig } from '../endpoint-client'
+import { Endpoint } from '../endpoint'
+
+
+export type SchemaAppId = {
+	schemaAppId: string
+}
+
+export type SchemaAppInvitationId = {
+	invitationId: string
+}
+
+export type SchemaAppInvitationSummary = {
+	invitationId: string
+	acceptUrl: string
+}
+
+export type SchemaAppInvitationCreate = SchemaAppId & {
+	description?: string
+}
+
+export type SchemaAppInvitation = SchemaAppId & {
+	id: string
+	description?: string
+	expiration?: number
+	acceptUrl?: string
+	declineUrl?: string
+	shortCode?: string
+}
+
+export type SchemaAppAcceptanceStatus = Omit<SchemaAppInvitation, 'id'> & {
+	isAccepted?: boolean
+}
+
+
+export class InvitesEndpoint extends Endpoint {
+	constructor(config: EndpointClientConfig) {
+		super(new EndpointClient('invites', config))
+	}
+
+	public async createSchema(schemaAppInvitation: SchemaAppInvitationCreate): Promise<SchemaAppInvitationId> {
+		return this.client.post('schemaApp', schemaAppInvitation)
+	}
+
+	public async listSchema(schemaAppId: string): Promise<SchemaAppInvitation[]> {
+		return this.client.getPagedItems('schemaApp', { schemaAppId })
+	}
+
+	public async revokeSchema(invitationId: string): Promise<void> {
+		await this.client.delete(`schemaApp/${invitationId}`)
+	}
+
+	public async getAcceptanceStatus(invitationId: string): Promise<SchemaAppAcceptanceStatus> {
+		return this.client.get('schemaApp/checkAcceptance', { invitationId })
+	}
+
+	public async acceptSchema(shortCode: string): Promise<SchemaAppId> {
+		return this.client.put(`schemaApp/${shortCode}`)
```

Comment:
I think this might need to PUT to `schemaApp/${shortCode}/accept` rather than just `schemaApp/${shortCode}`.

Reply:
Oops, fixed.